### ISSUE TITLE
training time

### ISSUE THREAD

**Issue by thepark96** [roytseng-tw/px2graph_lab#1]
How much epoch did this model take?
 And isnt the loss diverge when you use learning rate 3e-3?
